saleor/tests/fixtures.py

```diff
@@ -4129,8 +4129,6 @@ def draft_order_with_preorder_lines(order_with_preorder_lines):
     return order_with_preorder_lines
 
 
-# FIXME This fixtures should be renamed to legacy_ when we will convert current payment
-# flow to depreceated
 @pytest.fixture
 def payment_txn_preauth(order_with_lines, payment_dummy):
```

Comment:
But why not deprecated?

Reply:
We changed an approach here. Previously I assumed that we will convert model `Payment` to `LegacyPayment` but we don't get any benefits about renaming it for now.
This fixme was added to keep in mind about this, but with separate model that we created `TransactionItem`. It is not needed anymore.